cli/cli/commands/cloud/load/load.go

```diff
@@ -50,6 +51,18 @@ func run(ctx context.Context, _ *flags.ParsedFlags, args *args.ParsedArgs) error
 		return stacktrace.Propagate(err, "Expected a value for instance id arg '%v' but none was found; "+
 			"this is a bug in the Kurtosis CLI!", instanceIdentifierArgKey)
 	}
+
+	contextsConfigStore := store.GetContextsConfigStore()
+	currentContext, err := contextsConfigStore.GetCurrentContext()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while retrieving the current context")
+	}
+
+	if currentContext.Uuid.Value == instanceID {
+		logrus.Infof("Cloud instance %s already loaded", instanceID)
+		return nil
+	}
```

Comment:
Make the cloud load command idempotent.

Reply:
nice!

Reply:
Is it possible you might need to reload the config? If you do I guess you have to jus switch to default and back to this remote context?

Reply:
Yes and you cannot reload today, it fails on the remove context operation saying: "you cannot remove the current context."  More work would be required to reload.